sgt-back-end/queries.js

```diff
@@ -0,0 +1,17 @@
+const doesGradeIdExist = 'SELECT * FROM "grades" WHERE "gradeId" = $1';
```

Comment:
Casing on this is inconsistent with the others. If you want to have a constant that's all uppercase, you can use `_` to separate words: `DOES_GRADE_ID_EXIST`.